ui/src/app/pages/pages-routing.module.ts

```diff
@@ -1,234 +1,135 @@
-import {RouterModule, Routes} from '@angular/router';
```

Comment:
Please, in the routes check if these still exist, and makes sensor for the ORB project, if not please remove them. also, this logic where we have a few routes for production and other ones for development, won't be necessary for the time being.